ci/official/README.md

```diff
@@ -1,295 +1,131 @@
-
-1.  `wheel.sh` builds the TensorFlow Pip package and verifies its contents.
-2.  `pycpp.sh` runs an extensive `bazel test` suite whose targets vary depending
-    on the platform (target selection is handled in TensorFlow's bazelrc)
-3.  `libtensorflow.sh` builds
-    [libtensorflow](https://www.tensorflow.org/install/lang_c).
-4.  `code_check_full.sh` and `code_check_changed_files.sh` run some static code
-    analysis checks.
-
-Our CI runs these under a variety of environments that will receive additional
-documentation in the future.
-
-### Running Tests Yourself
-
-To run tests yourself, you'll copy the `envs/sample` file, adjust it to match
-your environment, `export TFCI=your-path`, and then simply run the script you
-want. A complete example is below this explanation. Some tips:
-
--   We generally use `<tensorflow-directory>/build_output` for all temporary
-    storage and build artifacts, and you'll find all output files there,
-    including `script.log`, the log of the last executed build script.
--   You can find out which env file a TensorFlow job used by looking at the
-    `TFCI` variable in the `BUILD_CONFIG` section of the Invocation Details for
-    that job, either in Sponge (internal) or ResultStore (external).
--   The files in `envs/` are configured to match TensorFlow's CI system and
-    reference paths and settings. They will not work out-of-the-box, so you'll
-    need to copy `sample` instead, which removes all of those custom details.
--   `sample` also resets the Python version to TensorFlow's default. You can
-    target a specific version by providing e.g.
-    `--repo_env=TF_PYTHON_VERSION=3.10` in `TFCI_BAZEL_COMMON_ARGS` as the other
-    `env` files do.
-
-Here is a complete example of how to set up and run a script:
-
-```
-cd <tensorflow-root-directory>
-mkdir -p build_output
-cp ci/official/envs/sample build_output/env
-vim build_output/env  # update "your_choice_here" to a real path
-export TFCI=$(realpath build_output/env)
-./ci/official/wheel.sh
+## TensorFlow's Official CI and Build/Test Scripts
+
+TensorFlow's official CI jobs run the scripts in this folder. Our internal CI
+system, Kokoro, schedules our CI jobs by combining a build script with an `env` file that is filled with configuration options:
+
+- Nightly jobs (Run nightly on the `nightly` branch)
+    - Uses `wheel.sh`, `libtensorflow.sh`, `code_check_full.sh`
+    - Uses `envs/nightly_...`
+- Continuous jobs (Run on every GitHub commit)
+    - Uses `pycpp.sh`
+    - Uses `envs/continuous_...`
+- Presubmit jobs (Run on every GitHub PR)
+    - Uses `pycpp.sh`, `code_check_changed_files.sh`
+    - Also uses `envs/continuous_...`
+
+These `envs` match up with an environment matrix that roughly covers:
+
+-   Linux, MacOS, and Windows machines (these pool definitions are internal)
+-   x86 and arm64
+-   CPU-only, or with NVIDIA CUDA support (Linux only), or with TPUs
+-   Different Python versions
```

Comment:
WDYT about making this just an explicit table?  This table on JAX is what made me think about it https://github.com/google/jax?tab=readme-ov-file#supported-platforms

Reply:
Sure, but let's punt the table for later, since some of the jobs right now are WIP.